tests/test_key_revocation.py

```diff
@@ -385,9 +388,6 @@ def test_root_key_revocation(self):
       for mirror_exception in exception.mirror_errors.values():
         self.assertTrue(isinstance(mirror_exception, tuf.BadSignatureError))
 
-    # Load the previous Root signing key so that the the client can update
-    # successfully.
-    repository.root.load_signing_key(self.role_keys['root']['private'])
```

Comment:
I am in the process of integrating this branch with some design changes and resolving merge conflicts.  I'll try take care of any remaining issues and let you know of any problems that remain.